.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,108 @@
+name: Bug report
+description: File a bug report. If you need help, contact support instead
+labels: [needs-triage, bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please check if your bug is [already filed](https://github.com/ollama/ollama/issues) before filing a new one.
```

Comment:
Client "server EOF" errors are ripe for pile-ons.  Should we try to capture that here (I worry that might be too in the weeds but ...)

Reply:
I'm not clear on what you're asking for. Let's chat soon and open another PR for this?